R/annotate.R

```diff
@@ -0,0 +1,52 @@
+# Annotate Graphs (should this go into gGtools.R?)
+#
+#' Produce annotated gene graph where vertices have scores
+#' attributed to proper genes.
+#'
```

Comment:
where all the IDs need to have been converted into HGNC format with `fastMap`, yes?